### ISSUE TITLE
Assets aren't being added

### ISSUE THREAD

**Issue by hkrol** [variar/grav-plugin-unitegallery#8]
I'm using Grav 1.2.3 and unitegallery plugin v1.1.2.
No JS and CSS assets are being added to the page. Despite switching off the caches.
I'm not using your modular sample, just directly on one page:

```twig
{% extends 'partials/base.html.twig' %}
{% block content %}
    {{ page.content }}

    <div class="modular-row gallery-container {{ page.header.class }}">
	{{ unite_gallery(page.media.images) }}
    </div>
{% endblock %}
```

Does this matter? The html is being generated correctly by the plugin.
I poked around the source a bit and it doesn't seem that the addJS and addCSS Methods do anything inside the addAssets function. The $jsAssets paths are correct.

**Comment by variar** [comment]
Hi, could you please add your base.html.twig? I have one idea why assets are missing and  need it to verify

**Comment by hkrol** [comment]
Thank you for looking into this. This is my simple base.html.twig:

```twig
<!DOCTYPE html>
<html lang="en">
<head>
    {% block head %}
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        {% if header.description %}
            <meta name="description" content="{{ header.description }}">
        {% else %}
            <meta name="description" content="{{ site.description }}">
        {% endif %}
        {% if header.robots %}
            <meta name="robots" content="{{ header.robots }}">
        {% endif %}
        
        <link rel="icon" type="image/png" href="{{ theme_url }}/images/favicon.png">

        <title>{% if header.title %}{{ header.title }} | {% endif %}{{ site.title }}</title>

        {% block stylesheets %}
            {% do assets.add('https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css',101) %}
            {% do assets.add('theme://css/milligram.css',101) %}
            {% do assets.add('theme://css/scss/style.css',101) %}
            {% do assets.add('theme://css/font-awesome.min.css',101) %}
        {% endblock %}
        {{ assets.css() }}

        {% block javascripts %}
            {% do assets.addJs('jquery', 100) %}
        {% endblock %}
        {{ assets.js() }}

    {% endblock head %}
</head>

<body>
    <main class="wrapper">

        {% include 'partials/navigation.html.twig' %}

        <div class="container">
            {% block content %}{% endblock %}
        </div>

        <footer class="footer">
            <section class="container">
                <hr>
                <p>&copy;2017 HOFER + KROL GbR</p>
            </section>
        </footer>
    </main>
</body>
</html>```

**Comment by variar** [comment]
Seems that modular and non-modular page processing differs somehow.

**Comment by variar** [comment]
Added crude fix in 1.1.3 version and some documentation. Please check it out.

**Comment by variar** [comment]
Created a demo site with both modular and non-modular examples: https://github.com/variar/grav_example_unitegallery
